packages/rich-text/src/component/index.js

```diff
@@ -325,8 +325,32 @@ function RichText(
 			multilineTag,
 			preserveWhiteSpace,
 		} );
-		event.clipboardData.setData( 'text/plain', plainText );
-		event.clipboardData.setData( 'text/html', html );
+
+		const isLi = /<li>.*<\/li>/gi;
+		const foundLi = html.match( isLi );
+		if ( foundLi ) {
+			let liCopyString = html
+				.split( '<li>' )
+				.join( '' )
+				.split( '</li>' )
+				.filter( Boolean );
+			const liCopyStringLength = liCopyString.length - 1;
+			liCopyString = liCopyString.map( ( el, i ) => {
+				if ( el && i === 0 ) {
+					return '<ul><li>' + el + '</li>';
+				} else if ( el && i === liCopyStringLength ) {
+					return '<div style="clear:both"/><li>' + el + '</li></ul>';
+				}
+				return '<div style="clear:both"/><li>' + el + '</li>';
```

Comment:
if `el` could be falsy (in two `if` statements above we check for it) and would be for the first element, then it will fall to the last case `'<div style="clear:both"/><li>' + el + '</li>'` what would generate improper HTML ( `<ul>` will be missing)

Reply:
I added a check for last return in my commit , but my lint checker log me an error :
"
  337:59  error  Array.prototype.map() expects a value to be returned at the end of arrow function  array-callback-return

"
It seems it must be some return in the end of function , or I can disable error with comment

Reply:
If it is possible that `el` is empty why not filter them out before mapping?

Reply:
i ve filter all elements in .filter( Boolean ) function before mapping